src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add: function (num) {
+      this.result += num;
+
+      return this;
+    },
+    subtract: function (num) {
+      this.result -= num;
+
+      return this;
+    },
+    multiply: function (num) {
+      this.result *= num;
+
+      return this;
+    },
+    divide: function (num) {
+      this.result /= num;
+
+      return this;
+    },
+    reset: function () {
+      this.result = 0;
+
+      return this;
+    },
+
+    operate: function (operation, num) {
+      operation.call(this, num);
+
+      return this;
```

Comment:
[CODE KNOWLEDGE] - Consistency in code is key. Use the method shorthand syntax for the `operate` method as well.

Replace:
operate: function(operation, num) {
  // your code
},

With:
operate(operation, num) {
  // your code
},